modules/registry-change-management-state.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_restricted_networks/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_restricted_networks/installing-restricted-networks-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+// * registry/configuring-registry-storage/configuring-registry-storage-baremetal.adoc
+// * registry/configuring-registry-storage/configuring-registry-storage-vsphere.adoc
+
+[id="registry-change-management-state_{context}"]
+= Change image registry ManagementState
+
+To start the image registry, you must change `ManagementState` Image Registry Operator configuration from `Removed` to `Managed`.
+
+.Procedure
+
+* Change `ManagementState` Image Registry Operator configuration from `Removed` to `Managed`. For example:
++
+[source,yaml]
+----
+apiVersion: imageregistry.operator.openshift.io/v1
+kind: Config
+metadata:
+  creationTimestamp: <time>
+  finalizers:
+    - imageregistry.operator.openshift.io/finalizer
+  generation: 3
+  name: cluster
+  resourceVersion:  <version>
+  selfLink: <link>
+spec:
+  readOnly: false
+  disableRedirect: false
+  requests:
+    read:
+      maxInQueue: 0
+      maxRunning: 0
+      maxWaitInQueue: 0s
+    write:
+      maxInQueue: 0
+      maxRunning: 0
+      maxWaitInQueue: 0s
+  defaultRoute: true
+  managementState: Managed 
+  ...
```

Comment:
Not sure you need the ellipsis in the yaml file? In our other docs, I see it in command code blocks only, though my search was minimal.

Reply:
Updated.